tests/test_generics.py

```diff
@@ -853,6 +853,17 @@ class Model(GenericModel, Generic[T]):
         # example)
         assert replace_types(list[Union[str, list, T]], {T: int}) == list[Union[str, list, int]]
 
+    if sys.version_info >= (3, 10):
```

Comment:
separate test with `skipif` please.

Reply:
also, please can you add a standalone test for unions on a generic model.

There should be examples elsewhere of testing `|` style unions.

Reply:
> separate test with `skipif` please.

done



> also, please can you add a standalone test for unions on a generic model.
> 
> There should be examples elsewhere of testing `|` style unions.

kinda done though i don't have a feeling that that's enough and might add some more later.

Reply:
sounds good, let me know when it's done.